webapp-client/src/App.js

```diff
@@ -11,24 +12,17 @@ export default class App extends React.Component {
     // hard-coded for now, will be fetched from db
     this.defaultCategory = 'All';
     this.state = {
-      // list of category names for now, will be list of category ids later
-      categories: [this.defaultCategory],
-      currentCategory: this.defaultCategory,
+      // categoryID of the current category
+      categoryId: this.defaultCategory,
+      categoryTitle: this.defaultCategory,
```

Comment:
Is "All" ok to be used as the categoryId? Would it cause a type mapping error with ObjectId in the backend?

Reply:
When Anthony renders items, he has a conditional check, so if it's All then everything is fetched and there won't be an error